### PR TITLE
User profile

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by RoxVer** [comment]
Reviewed

File: src/main/java/ua/softserve/rv_018/greentourism/model/Place.java [hunk 1]
```diff
@@ -35,6 +35,9 @@
 	@ManyToOne(cascade = CascadeType.ALL)
 	@JoinColumn(name = "category_id")
 	private Category category;
+	
+	@Column(name = "place_foto")
```
Comment (KostyaTk) [inline src/main/java/ua/softserve/rv_018/greentourism/model/Place.java]: What's this? If it meant to be a picture of the place, we have Gallery table for it.

File: src/main/java/ua/softserve/rv_018/greentourism/model/User.java [hunk 2]
```diff
@@ -25,6 +25,10 @@
     private String password;
     @Column(name = "is_active", nullable = false, columnDefinition = "boolean default false")
     private boolean isActive;
+    @Column(name = "social_account")
+    private String socialAccount;
+    @Column(name = "userpic")
```
Comment (KostyaTk) [inline src/main/java/ua/softserve/rv_018/greentourism/model/User.java:30]: Same as with place pic.

File: src/main/java/ua/softserve/rv_018/greentourism/model/Place.java (hunk 1 above)
Comment (47souls) [reply src/main/java/ua/softserve/rv_018/greentourism/model/Place.java]: So it must be a reference another field annotation in this field?

**Comment by ghost** [comment]
Thanks,  i didn't notice gallery table from the beginning, and forgot about it. I'll fix it after lesson.

File: src/main/webapp/resources/app/routes.js [hunk 3]
```diff
@@ -3,11 +3,13 @@
...
 		template : '<login></login>'
-	}).when('/profile', {
+	}).when('/profile/:id', {
 		template : '<profile></profile>'
 	}).when('/user/:userId', {
 		template : '<user></user>'
 	}).when('/map', {
 		template : '<map></map>'
+	}).when('/map/:id', {
+		template : '<map></map>'
```
Comment (Bmaks-github) [inline src/main/webapp/resources/app/routes.js:13]: You can use angular optional route parameters. If you want you can rewrite this. Here is a link: https://docs.angularjs.org/api/ngRoute/provider/$routeProvider

File: src/main/java/ua/softserve/rv_018/greentourism/model/User.java (hunk 2 above)
Comment (ghost) [reply src/main/java/ua/softserve/rv_018/greentourism/model/User.java:30]: Fix previous mistake. Created additional model as Attachment and Gallery where we can store some pictures.
userpic stays here, Misha told it's ok.

**Pull request merged**